spec/rails_helper.rb

```diff
@@ -7,18 +7,6 @@
 require 'rspec/rails'
 # Add additional requires below this line. Rails is not loaded until this point!
 
-# require database cleaner at the top level
-require 'database_cleaner'
-
-# configure shoulda matchers to use rspec as the test framework and full matcher libraries for rails
-Shoulda::Matchers.configure do |config|
```

Comment:
This file first started using a different tutorial, so now I am changing it to the tutorial i have chosen to use. See the link in the Pull Request description